data/windows.yaml

```diff
@@ -3347,3 +3360,104 @@ sources:
+  attackers to download arbitrary files from the internet, such as
+  "certutil.exe".
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%environ_systemroot%%\System32\config\systemprofile\AppData\LocalLow\Microsoft\CryptnetUrlCache\MetaData\*'
+    - '%%environ_systemroot%%\SysWOW64\config\systemprofile\AppData\LocalLow\Microsoft\CryptnetUrlCache\MetaData\*'
+    - '%%users.localappdata_low%%\Microsoft\CryptnetUrlCache\MetaData\*'
+    separator: '\'
+labels: [System, Network]
+conditions: [os_major_version >= 6]
+supported_os: [Windows]
+urls:
+- 'https://u0041.co/blog/post/3'
+- 'https://thinkdfir.com/2020/07/30/certutil-download-artefacts/amp/'
+- 'https://blog.menasec.net/2019/03/initial-access-execution-evidences-for.html'
+- 'https://github.com/Velocidex/velociraptor/blob/32ff014a4504e95c2650fe4770bfda550d998884/artifacts/definitions/Windows/Forensics/CertUtil.yaml'
+---
+name: WindowsCryptnetUrlCacheContent
+doc: |
+  Content of a Windows cache of files downloaded from the internet.
+
+  Helpful when investigating the use of "Living of the Land" tools that allow
+  attackers to download arbitrary files from the internet, such as
+  "certutil.exe".
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%environ_systemroot%%\System32\config\systemprofile\AppData\LocalLow\Microsoft\CryptnetUrlCache\Content\*'
+    - '%%environ_systemroot%%\SysWOW64\config\systemprofile\AppData\LocalLow\Microsoft\CryptnetUrlCache\Content\*'
+    - '%%users.localappdata_low%%\Microsoft\CryptnetUrlCache\Content\*'
+    separator: '\'
+labels: [System, Network]
+conditions: [os_major_version >= 6]
+supported_os: [Windows]
+urls:
+- 'https://u0041.co/blog/post/3'
+- 'https://thinkdfir.com/2020/07/30/certutil-download-artefacts/amp/'
+- 'https://blog.menasec.net/2019/03/initial-access-execution-evidences-for.html'
+- 'https://github.com/Velocidex/velociraptor/blob/32ff014a4504e95c2650fe4770bfda550d998884/artifacts/definitions/Windows/Forensics/CertUtil.yaml'
+---
+name: WindowsStartupInfo
+doc: |
+  StartupInfo XML files.
+
+  The files include the user's SID in the name and there could be up to 5 per
+  user. They contain a list of processes that were executed within the first 90
+  seconds from the time the user logged in. The info includes start time, the
+  full command line and the parent process info, among other things.
+sources:
+- type: FILE
+  attributes:
+    paths: ['%%environ_systemroot%%\System32\WDI\LogFiles\StartupInfo\*.xml']
+    separator: '\'
+labels: [Execution]
+conditions: [os_major_version >= 6 AND os_minor_version >= 2]
+supported_os: [Windows]
+urls:
+- 'https://medium.com/dfir-dudes/startupinfo-autoruns-served-up-on-a-plate-ba2da0c753c5'
+- 'https://gist.github.com/mkorman90/272dbc126409135018de3210dd3d2eca'
+- 'https://www.trustedsec.com/blog/who-left-the-backdoor-open-using-startupinfo-for-the-win/'
+---
+name: WindowsCrashDumps
+doc: |
+  Windows Error Reporting (WER) files & crash dumps.
```

Comment:
`&` => and

Reply:
> Validation check is failing:
> 
> ```
> WARNING:root:Unsupported "%%users.localappdata_low%%" in path: %%users.localappdata_low%%\Microsoft\CryptnetUrlCache\MetaData\* defined by artifact definition: WindowsCryptnetUrlCacheMetadata in file: data/windows.yaml
> WARNING:root:Artifact definition: WindowsCryptnetUrlCacheMetadata in file: data/windows.yaml contains Windows path that contains an unsupported users variable: "%%users.localappdata_low%%". 
> WARNING:root:Unsupported "%%users.localappdata_low%%" in path: %%users.localappdata_low%%\Microsoft\CryptnetUrlCache\Content\* defined by artifact definition: WindowsCryptnetUrlCacheContent in file: data/windows.yaml
> WARNING:root:Artifact definition: WindowsCryptnetUrlCacheContent in file: data/windows.yaml contains Windows path that contains an unsupported users variable: "%%users.localappdata_low%%". 
> ```

That's really weird because `users.localappdata_low` is defined in https://github.com/google/grr/blob/master/grr/proto/grr_response_proto/knowledge_base.proto#L173. I'd be happy to replace it with a longer path, but is this error correct and this variable shouldn't be used?

Reply:
Done